_includes/sections/events.html

```diff
@@ -0,0 +1,90 @@
+{% comment %} 
+<!--
+  Connect Section
+
+  This partial is included on any page that should include social sharing info.
+  Currently used on the Contact page and is using the footer data file to get
+  social sharing information.
+  
+--> 
+{% endcomment %}
+
+{% assign events = site.sections |where:"type","event" |sort: "date" %}
+
+<section>
+  <div class="section">
+    <div class="container clearfix">
+      <div class="row display-flex">
+
+        {% for event in events %}
+        
+          <div class="col-md-12 bottommargin">
+            <div class="clearfix">
+              <div class="event-image">
+                <img src="{{ site.baseurl }}{{ event.image }}" alt="Event">
+                <div class="event-date">06<span>May</span></div>
```

Comment:
Hard coded content.

Reply:
This is fixed now in 4f80ac0dc1dff3c90815d53ea14d7116661745ff